docs/ota-client-guide/modules/ROOT/pages/client-provisioning-methods.adoc

```diff
@@ -14,63 +14,58 @@ endif::[]
+* You should download your initial provisioning key from your OTA Connect account, and bake it into the image for the device.
+* The first time the device comes online, it connects to the device gateway and says, “Hi, I’m a new device, and I don’t have a TLS certificate yet. Here’s my provisioning key.”
+* Then, our crypto service generates a new keypair and X.509 certificate for the device, signs that certificate with the private fleet root CA in the Vault instance, and sends the whole bundle to the vehicle.
+* Lastly, it deletes the vehicle’s private key.
 
-This method is fine for provisioning devices quickly but if a malicious actor steals your provisioning key, there's no way to prevent unauthorized devices from provisioning. You'd have to blacklist the provisioning key for all devices and issue a new one.
+This entire transaction is secured with TLS, using the pinned device gateway certificate provisioned in the image.
 
-== Provisioning with device credentials
+
+
+=== Provisioning with device credentials
 
 If you're using OTA Connect in production, you should provision with device credentials.
-In this scenario, the OTA Connect server doesn't issue any credentials to devices. You need to preinstall the device certificates yourself.
-How you get the certificates on to your devices is your decision. Your chosen method can depend on many variables such as the storage method on the device, whether you have connectivity at your factory, and whether you choose an internal or third party root of trust for your certificates.
-For example, you might have a deployment process where your build system requests a certificate from your CA while a software image is being built. The device certificate is then installed on the image just before it is flashed to the target device.
-Once a device boots up, it connects to the OTA Connect server and provides the pre-installed device certificate to verify the device identity. The OTA Connect server then verifies that the certificate came from your CA.
-With this method, it is extremely difficult for an unauthorized device to join your fleet.
+In this method there is no shared credential. Instead the following steps occur:
 
+* You provide us with your own fleet root CA’s certificate (but NOT the private key).
+* Then, you make sure that each device acquires, in some out-of-band process, an X.509 certificate signed by your fleet root CA. In an ideal world, you would generate a keypair and a self-signed X.509 certificate inside an HSM on the device (so that there’s never any private key material outside of the HSM), then submit a CSR for that self-signed device certificate using your own existing PKI (e.g. via PKCS#10), and an authentication method of your choosing. (In an automotive OEM context, for example, that might be a private server inside the factory infrastructure, using the physical location and an airgapped network as ways to authenticate the request’s validity.)
```

Comment:
I think the parentheses and one of the commas are unnecessary.
```suggestion
* Then, you make sure that each device acquires, in some out-of-band process, an X.509 certificate signed by your fleet root CA. In an ideal world, you would generate a keypair and a self-signed X.509 certificate inside an HSM on the device (so that there’s never any private key material outside of the HSM), then submit a CSR for that self-signed device certificate using your own existing PKI (e.g. via PKCS#10) and an authentication method of your choosing. In an automotive OEM context, for example, that might be a private server inside the factory infrastructure, using the physical location and an airgapped network as ways to authenticate the request’s validity.
```